packages/shared/src/components/comments/CommentContainer.tsx

```diff
@@ -145,8 +149,14 @@ export default function CommentContainer({
                 @{comment.author.username}
               </TruncateText>
             </ProfileLink>
-            <div className="mx-2 h-0.5 w-0.5 bg-text-quaternary" />
+            <Separator />
             <CommentPublishDate comment={comment} />
+            {topReader && (
+              <>
+                <Separator />
+                <TopReaderIn topReader={topReader} tooltip />
```

Comment:
We should check how this behave with accessibility, now aria-label state "November 1996", but the actual content is more meaningful!

<img width="391" alt="image" src="https://github.com/user-attachments/assets/b010b137-6101-407f-9cf9-2c1641b83f45">

Reply:
It hardcodes the `aria-label` to be the content
```ts
tooltipProps['aria-label'] = content
```

Reply:
Yes, maybe we can add `aria-label` as`TopReaderIn` prop and (later even in a new PR) set that if `aria-label` is already set we don't want to override it?

Reply:
It's from the tooltip

Reply:
But I am extending the tooltip component to take aria-label prop to use instead.

Reply:
Nice, thanks!